napari_animation/_tests/test_animation.py

```diff
@@ -1,26 +1,90 @@
-import pytest
-from napari_animation import Animation
-
+from unittest.mock import patch
 
-@pytest.fixture
-def make_napari_viewer(qtbot, request):
-    from napari import Viewer
-    viewers = []
-
-    def actual_factory(*model_args, viewer_class=Viewer, **model_kwargs):
-        model_kwargs.setdefault('show', False)
-        viewer = viewer_class(*model_args, **model_kwargs)
-        viewers.append(viewer)
-        return viewer
+import numpy as np
+import pytest
 
-    yield actual_factory
+from napari_animation import Animation
 
-    for viewer in viewers:
-        viewer.close() 
+from .fixtures import make_napari_viewer, empty_animation, animation_with_keyframes, viewer_state
```

Comment:
fyi: the "standard" way to use fixtures is either to declare them directly in the file in which you need to use them, or put them into a [conftest.py file](https://docs.pytest.org/en/stable/fixture.html#conftest-py-sharing-fixtures-across-multiple-files) for use across multiple files.  (this way _seems_ to work, but you almost never see it for some reason, and pytest doesn't document it?)

Reply:
this is really good to know - I guess sticking to the way things should be done does have its benefits - will fix up!